ibm/service/power/data_source_ibm_pi_dhcps.go

```diff
@@ -7,101 +7,89 @@ import (
 	"context"
 	"log"
 
-	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
-/*
-Datasource to get the list of dhcp servers in a power instance
-*/
-
+// Datasource to list dhcp servers in a power instance
 func DataSourceIBMPIDhcps() *schema.Resource {
-
 	return &schema.Resource{
 		ReadContext: dataSourceIBMPIDhcpServersRead,
 		Schema: map[string]*schema.Schema{
-
-			// Required Arguments
+			// Arguments
 			Arg_CloudInstanceID: {
-				Type:         schema.TypeString,
+				Description:  "The GUID of the service instance associated with an account.",
 				Required:     true,
+				Type:         schema.TypeString,
 				ValidateFunc: validation.NoZeroValues,
 			},
 
 			// Attributes
 			Attr_DhcpServers: {
-				Type:        schema.TypeList,
 				Computed:    true,
-				Description: "The list of all the DHCP Servers",
+				Description: "List of all the DHCP Servers.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						Attr_DhcpID: {
-							Type:        schema.TypeString,
 							Computed:    true,
-							Description: "The ID of the DHCP Server",
+							Deprecated:  "The field is deprecated, use id instead.",
```

Comment:
We do not want to deprecate this. This `dhcp_id` is needed.
The `id` attribute is a single, random guid.